src/utils/internal/getCallFrame.ts

```diff
@@ -2,25 +2,22 @@
  * Return the stack trace frame of a function's invocation.
  */
 export function getCallFrame() {
-  try {
-    const inspectionError = new Error()
-    inspectionError.name = 'Inspection Error'
-    throw inspectionError
-  } catch (error) {
-    const frames: string[] = error.stack.split('\n')
+  const frames: string[] = (new Error().stack as string).split('\n')
 
-    // Get the first frame that doesn't reference the library's internal trace.
-    // Assume that frame is the invocation frame.
-    const declarationFrame = frames.slice(1).find((frame) => {
-      return !/(node_modules)?\/lib\/(umd|esm)\//.test(frame)
-    })
+  // Get the first frame that doesn't reference the library's internal trace.
+  // Assume that frame is the invocation frame.
+  const rePathIsNotUseful = /(node_modules)?[\/\\]lib[\/\\](umd|esm)[\/\\]|^[^\/\\]*$/
```

Comment:
I was thinking about utilizing `path.resolve` and its built-in system-specific path delimiters. I suppose that won't work in a browser, will it?

Reply:
yeah, browsers use `webpack:///./test/msw-api/setup-worker/printHandlers.mocks.ts?:6:113` kind of path on all OS